setup.cfg

```diff
@@ -34,7 +34,7 @@ source = aiohttp, tests
 omit = site-packages
 
 [tool:pytest]
-addopts = --cov=aiohttp -v -rxXs --durations 10
+addopts = --cov=aiohttp -v -rxXs --durations 15
```

Comment:
we probably don't want to change this

Reply:
Yes, but it took so long to trigger the warnings so I tried to increase it.
It's back to normal now.